ConsoleTest/Program.cs

```diff
@@ -5,81 +5,69 @@
 using MailKit;
 using MimeKit;
 using MailKit.Net.Smtp;
+using System.Threading;
+using static System.Net.Mime.MediaTypeNames;
+using System.Diagnostics;
 
 namespace ConsoleTest
 {
+    public delegate void Callback(long count);
     class Program
     {
+        
+
         static void Main(string[] args)
         {
-            //SendMessageNetMail();
-            SendMessageMailKit();
-            Console.Read();
+            Console.Write("Введите число для рассчета факториала этого числа и суммы целых чисел от нуля до этого числа: ");
+            long num = Int64.Parse(Console.ReadLine());
```

Comment:
```suggestion
            long num = long.Parse(Console.ReadLine());
```
Int64 - глаз режет.
Int64 - это класс C#
long - это тип CLR. В принципе, одно и тоже.